javascript/funWithFunctions.md

```diff
@@ -0,0 +1,26 @@
+## Functional Expressions in D&D ##
+I've been having fun with functions while creating a JavaScript version of the 5th edition D&D character sheet.
+
+In recreating the character bonus table I got thru 47 lines of inline conditionals before the formula slapped me in the face:  
+![](http://images.thoughtbot.com/TIL/OG_s.jpg)
+
+
+Even in adding abilities to an array, I was still left with 6 variables and a declared function:
+
+```javascript
+var abilities = [9, 10, 15, 12, 9, 19];  //The [str, dex, con, int, wis, chr] of my current character
```

Comment:
This line is long and causes a scrollbar to be added to the code example in the rendered markdown. Thoughts on moving it to its own line above this? Alternatively, maybe just changing "of my current character" to "of character" to make this shorter?